### ISSUE TITLE
Wording Enhancement/Suggestions

### ISSUE THREAD

**Issue by Alkallid** [Amazinite/alphean-expansion#1]
Disiuze
Feel free to think about any of these
Intel Rocket
"...missiles, they..."
Semicolon or new sentence

For the Chaingun and Nebulon, and anywhere else, don't use numerals, use the English forms

Nebulon Turret suggestion:
This weapon gives you a pair of very accurate Nebulon Beams on an even more precise turret, ensuring that nobody is left out from death.

I don't think nebulon needs to be capitalized unless it's in a name

Snazzing the Grenadier, based on what the stats imply you want:
After seeing so many different missile designs that all focused on firing swarms of weak homing projectiles, <weapon manufacturer> took inspiration from an archaeic desgn- grenades. Instead of making the shot home towards its target, grenades are meant to be fired mid-range, and let the effective radius work for you.

**Comment by Alkallid** [comment]
"A newly developed weapon, this weapon lobs a high AP slug at an enemy ship in an effort to pierce the hull or even split it in half."

```
description "The heat this weapon generates could disable your own ship's system so it could be a good idea to wait for the perfect moment."
```

"This cannon could easily overheat your ship after firing, so it is highly recommended to find an exact, oppurtune moment to unleash the rail."
Also, you used weapon too repetitively
Anti-ship battery-
Considering it's a battery, you might want to cahnge a few things. For one, give it an icon, otherwise that Sparrow you wanted to kill with you Nebulon will give nuked and you'll be burning to death in your ship.
Another is to actually make it a battery by giving burst count 6 or 12, burst reload 5, and reload 300 or so. Also divide heat and energy to evenly cost. Then change 
 innacuracy so some shots land, maybe tenfold.
Maybe give it 50 blast radius, like shells striking a battleship. Increase velocity.
 4 is very accurate.
Just a suggestion.

Wording-wise, 
"This an anti-capital destroyer. It barrages large warships with repeated volleys of high-power shells, but since that is it's main purpose, it very easily misses small ships and strains your cooling systems."

"The EMP launcher launches EMPs." I DIDN'T KNOW THAT!

Did you forget to add velocity? Same for the Pulse.

"The Electromagnetic Pulse Bomb Launcher, EMPL for short, cuts off foes' energy generation, forcing them to rely on batteries until they lie helpless under heavy fire.

"This version of the EMPL contains several charges of ion bursts, so releasing one on even a capital nearly always puts it out of commission.

**Comment by Disiuze** [comment]
Alright, I'll see what I can do. Thanks!

**Comment by Disiuze** [comment]
Addressed in 2acbd45.
